ci/plugins/mzcompose/hooks/pre-exit

```diff
@@ -20,23 +20,19 @@ run() {
 }
 
 run logs --no-color > services.log
-buildkite-agent artifact upload services.log
-
 # shellcheck disable=SC2024
 sudo journalctl --merge > journalctl-merge.log
-buildkite-agent artifact upload journalctl-merge.log
-
 netstat -ant > netstat-ant.log
-buildkite-agent artifact upload netstat-ant.log
-
 netstat -panelot > netstat-panelot.log
-buildkite-agent artifact upload netstat-panelot.log
-
 ps aux > ps-aux.log
-buildkite-agent artifact upload ps-aux.log
-
 docker ps -a --no-trunc > docker-ps-a.log
-buildkite-agent artifact upload docker-ps-a.log
+
+ARTIFACTS=(run.log services.log journalctl-merge.log netstat-ant.log netstat-panelot.log ps-aux.log docker-ps-a.log)
+for ARTIFACT in "${ARTIFACTS[@]}"; do
+  buildkite-agent artifact upload "$ARTIFACT"
+done
+
+bin/ci-flake-detect --name=mzcompose "${ARTIFACTS[@]}"
```

Comment:
I think the `--name` here must be more specific than just `mzcompose`, e.g. it should come from the `BUILDKITE_STEP_KEY` variable?